test/library/standard/FileSystem/chmod/changeMode.execopts

```diff
@@ -0,0 +1,38 @@
+--permissions 0o0600 # changeMode.rwu.good
+--permissions 384 # changeMode.rwu.good
+--permissions 0o0400 # changeMode.ru.good
+--permissions 256 # changeMode.ru.good
+--permissions 0o0200 # changeMode.wu.good
+--permissions 128 # changeMode.wu.good
+--permissions 0o0100 # changeMode.xu.good
+--permissions 64 # changeMode.xu.good
+--permissions 0o0700 # changeMode.rwxu.good
+--permissions 448 # changeMode.rwxu.good
+--permissions 0o0040 # changeMode.rg.good
+--permissions 32 # changeMode.rg.good
+--permissions 0o0020 # changeMode.wg.good
+--permissions 16 # changeMode.wg.good
+--permissions 0o0010 # changeMode.xg.good
+--permissions 8 # changeMode.xg.good
+--permissions 0o0070 # changeMode.rwxg.good
+--permissions 56 # changeMode.rwxg.good
+--permissions 0o0004 # changeMode.ro.good
+--permissions 4 # changeMode.ro.good
+--permissions 0o0002 # changeMode.wo.good
+--permissions 2 # changeMode.wo.good
+--permissions 0o0001 # changeMode.xo.good
+--permissions 1 # changeMode.xo.good
+--permissions 0o0007 # changeMode.rwxo.good
+--permissions 7 # changeMode.rwxo.good
+--permissions 0o4000 # changeMode.su.good
+--permissions 2048 # changeMode.su.good
+--permissions 0o2000 # changeMode.sg.good
+--permissions 1024 # changeMode.sg.good
+--permissions 0o1000 # changeMode.to.good
+--permissions 512 # changeMode.to.good
```

Comment:
The above two lines failed on my system.  Instead of:
```
1000
1000
```
I get:
```
2000
2000
```

Reply:
I ran it using `start_test changeMode.chpl` and also by directly compiling it and running it by setting the `permissions` at runtime. For both, I get 1000. I am running Ubuntu. But I do not really think these are supposed to change with different versions of Linux. Could it be the sticky bit? It's giving you `2000` when you run it with both `2000` and `1000`?

Reply:
The umask setting in the `sub_test` file should avoid the sticky bit mattering, I would think.  At least, that's why we try to set it there.  And it looks like `sub_test` has the right permissions, so it should be getting executed.

Still musing

Reply:
What does an `ls -la` show for your current directory?  Mine shows this:
```
drwx--S---  3 <username> <group> 1378 Mar 29 15:46 ./
```

Reply:
Ah, the umask for that portion allows it to be defined by the system.  Hmm

Reply:
An `ls -la` on the current directory (`chmod`) shows:

> `drwxr-xr-x 3 <username> <group> 4096 Mar 28 23:19 ./`

`umask` shows: `0022`.

Reply:
Also, the `sub_test` file is meant to set the `umask` for the system, but where is it executed? Is it automatically done when `start_test` is called?